parsers/jscript.c

```diff
@@ -427,6 +427,17 @@ static void makeFunctionTag (tokenInfo *const token, vString *const signature, b
 	makeFunctionTagCommon (token, signature, generator, false);
 }
 
+static bool isClassName (tokenInfo *const name)
+{
+	char * p = strrchr(name->string->buffer, '.');
+	if (p == NULL)
+		p = name->string->buffer;
```

Comment:
Let's use vStringValue.
If you find lines that access `->buffer` directly in jscript.c (and the other files), they can be candidates for fixing in another pull request:-)